driver-scala/src/main/scala/org/mongodb/scala/gridfs/GridFSBucket.scala

```diff
@@ -170,6 +172,7 @@ case class GridFSBucket(private val wrapped: JGridFSBucket) {
    * @return a new GridFSBucket instance with the set time limit for operations
    * @since 4.x
```

Comment:
Shouldn't this be `@since CSOT` similarly to line `GridFSBucket.scala:124` for `timeout`?

Reply:
Yes, these placeholders are designed to streamline the release process, whether for versions 5.1 or 5.2. We could address them in a separate PR.